templates/release_checklist.md

```diff
@@ -0,0 +1,149 @@
+<!-- Checklist generated by release toolkit -->
+
+<!-- scheduled_release_only -->
+
+## Pre release
+
+- [ ] Clone or update the [release toolkit](https://href.li/?https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile) and review the [release script instructions](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] [Notifiy open Gutenberg Mobile PRs assigned to {{version_number}} milestone]({{milestone_url}}).
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+Hello!
+We will cut the {{version_number}} release on {{release_date}}.
+I plan to circle back and bump this PR to the next milestone then,
+but please let me know if you’d rather us work to include this PR in {{version_number}}.
+Thanks!
+```
+
+</p>
+</details>
+
+- [ ] Create WPiOS and WPAndroid builds from `develop`.
+- [ ] [Perform general writing flow tests on Android](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+- [ ] [Perform general writing flow tests on iOS](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+
+<!-- /scheduled_release_only -->
+
+## Create the Release ({{release_date}})
```

Comment:
Since we don't have a release date in beta and hotfixes, we need to have a different heading depending on the release type:

```suggestion
<!-- scheduled_release_only -->
## Create the Release ({{release_date}})
<!-- /scheduled_release_only -->
<!-- non_scheduled_release_only -->
## Create the Release
<!-- /non_scheduled_release_only -->
```